runtime/altair/src/lib.rs

```diff
@@ -652,16 +652,19 @@ parameter_types! {
 	pub CouncilMotionDuration: BlockNumber = prod_or_fast!(5 * DAYS, 1 * MINUTES, "AIR_MOTION_DURATION");
 	pub const CouncilMaxProposals: u32 = 100;
 	pub const CouncilMaxMembers: u32 = 100;
+	pub MaxProposalWeight: Weight = Perbill::from_percent(50) * RuntimeBlockWeights::get().max_block;
```

Comment:
**CLARIFICATION**

Does this also hold for runtime-upgrade proposals?

Reply:
@wischli leaving this for a post mortem review

Reply:
Will check on Monday!

Reply:
This is sufficient. It is an upper boundary on the weight of an proposal. As runtime ugprades usually have the `external_propose_majority` extrinsics as the thing voted on in the proposal the weight should be sufficient.